frontend/src/metabase/dashboard/components/DashboardHeader/DashboardHeaderButtonRow/DashboardHeaderButtonRow.tsx

```diff
@@ -77,7 +89,20 @@ export const DashboardHeaderButtonRow = ({
             ...props,
           };
 
-          if (config.enabled(buttonComponentProps)) {
+          if (
+            config.enabled({
```

Comment:
I couldn't find in `frontend/src/metabase/dashboard/components/DashboardHeader/DashboardHeaderButtonRow/action-buttons.tsx` that these props are used in `config.enabled`
- `refreshPeriod`
- `onRefreshPeriodChange`
- `setRefreshElapsedHook`
- `onFullscreenChange`
- `isNightMode`

We should remove any props that we don't use, otherwise, it's confusing.